src/compose/ComposeMenu.js

```diff
@@ -212,48 +232,28 @@ export default function ComposeMenu(props: Props): Node {
   const styles = useMemo(
     () =>
       createStyleSheet({
-        composeMenu: {
+        container: {
+          marginTop: 2, // So it doesn't run into the text input
+          borderWidth: 0,
+          borderBottomLeftRadius: 5,
+          borderBottomRightRadius: 5,
           flexDirection: 'row',
-          overflow: 'hidden',
-        },
-        expandButton: {
-          padding: 12,
-          color: BRAND_COLOR,
-        },
-        composeMenuButton: {
-          padding: 12,
-          marginRight: -8,
-          color: BRAND_COLOR,
+          backgroundColor,
         },
       }),
```

Comment:
![image](https://user-images.githubusercontent.com/28173/211864179-b9f34672-6eef-4a96-8ca4-e8fabd76af35.png)

I want to iterate a bit more on the design before we merge this. In particular, that thin horizontal line between the content box and the row of icons is jumping out at me — it gives me the feeling that something's glitched here.

Let's try leaving out this `backgroundColor` prop. Then we get something like the following:
![tmp](https://user-images.githubusercontent.com/28173/211866417-82150b03-3115-4ce1-b46c-7e2a486e0c92.png)
which I think looks a lot more plausible.

Two other things that look off to me when that `backgroundColor` is there:
* The row of icons looks a lot like the two text inputs — so it looks like it wants to be a third text input. But in fact it's quite different from a text input.
* There's a bunch of empty-looking space off to the right, where the background color keeps going past the icons it's there for.

Reply:
Sure, makes sense. I was looking at an old screenshot from Vlad on [CZO](https://chat.zulip.org/#narrow/stream/48-mobile/topic/UI.20redesign.3A.20Compose.20buttons/near/1472337), and trying to move slightly in the direction of the design there—while avoiding:
- complication in the diff
- having to pick a new color for the background of the buttons panel

<img width=400 src="https://user-images.githubusercontent.com/22248748/211877110-4cceeb63-bffb-4c8c-8a2f-079be9a36951.png" />

Your proposal successfully avoids those things, and I think also doesn't really get in the way of following Vlad's design in that screenshot (which was preliminary anyway and may change; [Vlad plans to open an issue](https://chat.zulip.org/#narrow/stream/48-mobile/topic/UI.20redesign.3A.20Compose.20buttons/near/1481618)). 🙂